src/UserApp.ts

```diff
@@ -2,7 +2,16 @@ import { DivsStructure, InputSubmitElems, Table } from "../utils/HtmlStructures"
 import { Btn, Space, HiddenDiv, TheadName, QueryResult, TbodyContent, QueryResultContent } from "../utils/HtmlElems"
 import { $, render, toLCaseIncludes, setSbmtBtnVis, setHiddenDivVis, toArr } from "./variables/functions"
 
-const updateState = (state: Array<any[]>) => {
+
+// Enum with members representing buttons on the top of the app
+enum CurrentButton {
+    Search,
+    Add,
+    Remove,
+    ShowAll
+}
+
+const updateState = (keyState: Array<Array<any>>, state: CurrentButton) => {
```

Comment:
updateState should only take a single argument, which is the State object. It can have two properties for your current use case: array of users, and the current button. 

Having state as a single object is a design most modern web apps follow, it's much cleaner and easier to track.